README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Apache Daffodil SBT Plugin
+
+Plugin to enable Daffodil capabilities for DFDL schema projects.
+
+## Enable
+
+To enable the plugin, add the following to `project/plugins.sbt`:
+
+```scala
+addSbtPlugin("org.apache.daffodil" % "sbt-daffodil" % "<version>")
```

Comment:
Interesting that the plugin's name (sbt-daffodil) is the reverse order of this repo's name (daffobil-sbt).  Should one be renamed to match the other's name?

Reply:
All sbt plugins have the naming convention of `sbt-foo` so I'm not sure we want to change the name here to "daffodil-sbt". But ASF requires that their repos start with the project name, so we can't name the repo "sbt-daffodil" like would be ideal. So the different in name/repo are because we're trying to follow two different conventions that we can't (and probably  shouldn't) really change.